python-threatexchange/threatexchange/extensions/vpdq/tests/test_vpdq_faiss.py

```diff
@@ -10,7 +10,7 @@
     import vpdq
 
     _DISABLED = False
-except (ImportError, ModuleNotFoundError) as e:
+except Exception:
```

Comment:
This is probably too generic, since it will catch any issue from vpdq, causing it to fail silently. 

We want to separate "expected" exceptions from "unexpected" exceptions.

Here are two solutions that would work better for me:
1. Throw ImportError, continue using import checks
2. Add test your exception class for a specific known argument so you know it's from the ffmpeg check, and not from anywhere else.

Reply:
Have checked to ImportError and specify only ImportError and ModuleNotFoundError.